src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,44 @@
+databaseChangeLog:
+  - changeSet:
+      id: 3
+      author: jitenvij
+      changes:
+        - dropTable:
+            tableName: person
+  - changeSet:
+      id: 4
+      author: jitenvij
+      changes:
+        - createTable:
+            tableName: person
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: first_name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: last_name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+  - changeSet:
+      id: 5
+      author: jitenvij1992
+      changes:
+        - insert:
+            tableName: person
+            columns:
+              - column:
+                  name: first_name
+                  value: Jiten
+              - column:
+                  name: last_name
+                  value: Vij
```

Comment:
I think it's better to keep ddl and `loading data` into separate files and include them in master file.

Reply:
yeah, I agree. I have changed the files